front50-pipelines/src/main/groovy/com/netflix/spinnaker/front50/notifications/NotificationRepository.groovy

```diff
@@ -76,39 +80,27 @@ class NotificationRepository {
         notifications
     }
 
-    Map getGlobal() {
+    Notification getGlobal() {
         get(HierarchicalLevel.GLOBAL, '')
     }
 
-    Map get(HierarchicalLevel level, String name) {
-        Map notification
+    Notification get(HierarchicalLevel level, String name) {
         try {
             Column<String> result = keyspace.prepareQuery(CF_NOTIFICATIONS)
                 .getKey(CF_NAME)
                 .getColumn("$level:$name".toString())
                 .execute().result
-            notification = mapper.readValue(Zip.decompress(result.byteArrayValue), Map)
+            return mapper.readValue(Zip.decompress(result.byteArrayValue), Notification)
         } catch (NotFoundException ignored) {
-            notification = [email: []]
-        }
-        if (level == HierarchicalLevel.APPLICATION) {
-            NOTIFICATION_FORMATS.each {
```

Comment:
This add `GLOBAL` on top of `APPLICATION` logic has been moved into the Controller such that both S3 and Cassandra can benefit.